base/src/main/java/com/nedap/archie/base/IntervalDurationConverter.java

```diff
@@ -0,0 +1,73 @@
+package com.nedap.archie.base;
+
+
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.time.temporal.TemporalAmount;
+import java.time.temporal.TemporalUnit;
+import java.time.temporal.UnsupportedTemporalTypeException;
+import java.util.Objects;
+
+import static java.time.temporal.ChronoUnit.DAYS;
+
+/**
+ * A tool to convert Durations from TemporalAmounts in the specific use case of Intervals
+ * It handles the unit Month as an exact number of seconds to do a conversion. This is a bit tricky, since
+ * a number of months has a lower number of seconds and an upper number of seconds. but we need some way of doing this...
+ *
+ * This may need a separate method for lower and upper, which returns the lower and upper bounds of the seconds in a month?
+ */
+class IntervalDurationConverter {
+
+//    private final static long MINIMUM_SECONDS_IN_DAY = 60*60*23;
+//    private final static long MINIMUM_SECONDS_IN_MONTH = 28*MINIMUM_SECONDS_IN_DAY;
+//    private final static long MINIMUM_SECONDS_IN_WEEK = 7*MINIMUM_SECONDS_IN_DAY;
+//
+//    private final static long MAXIMUM_SECONDS_IN_DAY = 60*60*25;
+//    private final static long MAXIMUM_SECONDS_IN_WEEK = 7*MAXIMUM_SECONDS_IN_DAY;
+//    private final static long MAXIMUM_SECONDS_IN_MONTH = 28*MAXIMUM_SECONDS_IN_WEEK;
+
+    public static Duration from(TemporalAmount amount) {
+        Objects.requireNonNull(amount, "amount");
```

Comment:
Maybe a more detailed error message, e.g.
```suggestion
        Objects.requireNonNull(amount, "amount must not be null");
```

Reply:
This is the exact same behaviour as Duration.from() except in the case of Estimated durations. That includes the error messages. Not changing it.